server/start.go

```diff
@@ -262,6 +262,7 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 			WithClient(local.New(tmNode))
```

Comment:
I know this isn't part of your change-set, but can you make this a one-liner please?